bin/client_package_standalone

```diff
@@ -4,19 +4,47 @@
 # Packaging script which creates standalone package.
 #
 # Usage:
-#> bin/client_package_standalone
+#> bin/client_package_standalone <os>
+#
+# Package (specific OS: linux, darwin, windows):
+#> bin/client_package_standalone linux
+#
+# Package (specific binary):
+#> BINARY=build/client/mysterium_client_darwin_amd64 bin/client_package_standalone darwin
 
 set -e
 
+source bin/helpers/output.sh
+
+OS=$1
+if [ -z "$OS" ]; then
+    print_error "Missing OS!"
+    exit 1
+fi
+if [ "$OS" != "linux" ] && [ "$OS" != "darwin" ] && [ "$OS" != "windows" ]; then
+    print_error "OS should be: linux, darwin, windows!"
```

Comment:
"OS should be: linux, darwin or windows", would skip "!".

Reply:
Improved